packages/node_modules/@ciscospark/widget-recents/src/container.js

```diff
@@ -313,55 +123,22 @@ export class RecentsWidget extends Component {
     } = this;
     const {
       sparkInstance,
-      spaces
+      spacesById
     } = props;
 
     let spaceId = activity.target && activity.target.id;
 
-    // Check for notification state update event(s) from a
-    // tag which contains "MESSAGE_NOTIFICATIONS_ON", "MESSAGE_NOTIFICATIONS_OFF",
-    // "MENTION_NOTIFICATIONS_ON", or "MENTION_NOTIFICATIONS_OFF"
-    const {tags} = activity.object;
-    const updateNotificationFeatures = tags && tags.size === 1 &&
-      (tags[0].inclues[MESSAGE_NOTIFICATIONS_ON]
-        || tags[0].includes[MESSAGE_NOTIFICATIONS_OFF]
-        || tags[0].includes[MENTION_NOTIFICATIONS_ON]
-        || tags[0].includes[MENTION_NOTIFICATIONS_OFF]);
-
-    if (updateNotificationFeatures) {
-      // Re-fetch group message notification feature
-      props.getFeature(FEATURES_USER, FEATURES_GROUP_MESSAGE_NOTIFICATIONS, sparkInstance).then(() => {
-        props.updateWidgetStatus({
-          hasFetchedGroupMessageNotificationFeature: true
-        });
-      });
-
-      // Re-fetch mention notification feature
-      props.getFeature(FEATURES_USER, FEATURES_MENTION_NOTIFICATIONS, sparkInstance).then(() => {
-        props.updateWidgetStatus({
-          hasFetchedMentionNotificationFeature: true
-        });
-      });
-    }
-
     // On delete, refetch space to get previous activity
     if (spaceId && ['delete', 'tombstone'].includes(activity.verb)) {
       props.fetchSpace(sparkInstance, spaceId);
     }
 
-    if (spaceId && ['post'].includes(activity.verb)) {
```

Comment:
This was to get space tags for muted state. We can get these via websocket and not 429 the convo server.